test/unit/http/auto_retry_test.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2015 Workiva Inc.
```

Comment:
With my suggestion to remove `calculate()` from the public API of `RetryBackOff` and into the common request implementation, these tests may not work as well. But I already have a set of integration tests for automatic retrying that gets run against every type of request here: https://github.com/Workiva/w_transport/blob/master/test/integration/http/common_request/suite.dart#L413-L482

I would recommend adding tests there.